src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -521,6 +521,19 @@ private function getColumnWidth($column)
                 continue;
             }
 
+            // split the TableCell centent in order to extending width of other columns
```

Comment:
Normally when we use `colspan`, the cells are merged into one and other cells are [left blank](https://github.com/aitboudad/symfony/blob/issues_18627/src/Symfony/Component/Console/Helper/Table.php#L441-L442), 
but when we want to calculate the column width we should split the merged cells content in different emptied cells in order to assume the average width of each columns